packages/remix/scripts/deleteSourcemaps.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+function deleteSourcemaps(buildPath) {
+  console.info(`[sentry] Deleting sourcemaps from ${buildPath}`);
+
+  const files = fs.readdirSync(buildPath);
```

Comment:
One more q: Is the output in the build folder always only one flat layer or could there be folders inside? Otherwise, we might need to recurse here, and/or use something like the `glob` library.

Reply:
Yes, it was missing out the files in `_shared` folder. Updated 👍